docs/partial_source/deep_dive/ivy_frontends.rst

```diff
@@ -581,9 +583,37 @@ An example function using this is the :func:`numpy.isfinite` function.
             ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
         return ret
 
+
+Frontends Duplicate Policy
+--------------------------
+Some frontend functions appear in multiple namespaces within the original framework that the frontend is replicating.
+For example the :func:`np.asarray` function appears in `Array manipulation routines`_ and also in `Array creation routines`_.
+Thus, rather than implementing these functions as duplicates in their respective namespaces in Ivy, this section outlines a policy
+that should serve as a guide for handling duplicate functions. The following sub-headings outline the policy:
+
+**Listing duplicate frontend functions on the ToDo lists**
+
+* if a function is duplicated across multiple namespaces then we should designate the function as an '(alias)' on each namespace list.
+This means that when creating ToDo lists, extra care should be taken to keep note of duplicate functions. Duplicate functions should have '(alias)' written beside the function name.
+
+**Contributing duplicate frontend functions**
+
+Before working on a frontend function, contributors should check if the function is designated as an alias on the ToDo list.
+If the function is an alias, you should check if there is an existing implementation already.
+
+* If an implementation exist then you should use the existing implementation as an alias in your implementation.
+* If there is no existing implementation, then feel free to contribute the function with its full implementation and tests.
+* If two PRs are implementing a duplicate function (and no alias for that function exists), then reviewers should select the most advanced (comprehensive) implementation to serve as the alias while the second implementation should be refactored to use the alias. Both contributors should be rewarded equally if need be.
+
+Tests should be written for duplicate fronend functions using aliases. However, when defining the :code:`fn_name` in the test, be sure to use the full import path to the function.
```

Comment:
could you please add a more concrete code example? Both aliases should be tested for in the same unit test. You should give the example of `np.asarray` as mentioned earlier.